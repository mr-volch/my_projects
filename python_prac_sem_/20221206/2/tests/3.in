import asyncio

async def main(A):
    tasks, B = await mtasks(A)
    random.shuffle(tasks)
    await asyncio.gather(*tasks)
    return B


random.seed(1091)
A = random.choices(range(25), k=128)
B = asyncio.run(main(A))
print(*A)
print(*B)
print(B == sorted(A))
