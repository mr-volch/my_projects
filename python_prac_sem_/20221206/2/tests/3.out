8 21 0 6 20 17 8 15 17 4 20 5 13 12 7 11 22 3 8 24 17 13 8 7 4 6 12 0 18 2 13 17 3 13 1 23 3 9 12 1 5 22 1 12 8 17 13 11 4 2 12 4 7 14 2 2 10 9 2 17 15 9 3 14 11 16 10 19 0 7 5 2 7 5 18 11 17 13 19 6 9 9 11 21 7 3 17 21 4 20 11 21 6 4 17 7 16 5 10 4 24 2 12 15 0 7 5 15 17 17 22 10 15 6 18 9 16 14 14 19 2 11 11 12 10 2 15 23
0 0 0 0 1 1 1 2 2 2 2 2 2 2 2 2 3 3 3 3 3 4 4 4 4 4 4 4 5 5 5 5 5 5 6 6 6 6 6 7 7 7 7 7 7 7 7 8 8 8 8 8 9 9 9 9 9 9 10 10 10 10 10 11 11 11 11 11 11 11 11 12 12 12 12 12 12 12 13 13 13 13 13 13 14 14 14 14 15 15 15 15 15 15 16 16 16 17 17 17 17 17 17 17 17 17 17 17 18 18 18 19 19 19 20 20 20 21 21 21 21 22 22 22 23 23 24 24
True
